web/cobrands/buckinghamshire/assets.js

```diff
@@ -172,6 +184,9 @@ fixmystreet.assets.add($.extend(true, {}, defaults, {
     all_categories: true,
     actions: {
         found: function(layer, feature) {
+            if (fixmystreet.body_overrides.get_only_send() === 'Highways England') {
```

Comment:
This causes a delay upon e.g. clicking from a motorway to a non-Bucks maintained road because this section fires first time (so nothing changes), highways layer removes the only_send, then when the server responds with updated point information it then skips this bit and disables the form. Perhaps this if isn't needed?

Reply:
And similarly if clicking from motorway to a Bucks road - only after the server response is it set up for e.g. Bucks-only for flytipping.